lib/spack/spack/package.py

```diff
@@ -137,8 +138,9 @@ def copy(self):
 
 
 class PackageMeta(
-    spack.directives.DirectiveMeta,
-    spack.mixins.PackageMixinsMeta
+        spack.directives.DirectiveMeta,
```

Comment:
Just curiosity: why double indentation here?

Reply:
This is what emacs did -- I moved it back in the new commit to minimize the diff.